types/three/examples/jsm/loaders/KTX2Loader.d.ts

```diff
@@ -1,17 +1,18 @@
-import { LoadingManager, CompressedTextureLoader, CompressedTexture, WebGLRenderer } from '../../../src/Three.js';
+import { LoadingManager, Loader, CompressedTexture, WebGLRenderer } from '../../../src/Three.js';
 import WebGPURenderer from '../renderers/webgpu/WebGPURenderer.js';
 
-export class KTX2Loader extends CompressedTextureLoader {
+export class KTX2Loader extends Loader {
     constructor(manager?: LoadingManager);
 
     setTranscoderPath(path: string): KTX2Loader;
     setWorkerLimit(limit: number): KTX2Loader;
     detectSupport(renderer: WebGLRenderer | WebGPURenderer): KTX2Loader;
     dispose(): KTX2Loader;
 
-    parse(
-        buffer: ArrayBuffer,
+    load(
+        url: string,
         onLoad: (texture: CompressedTexture) => void,
+        onProgress?: (event: ProgressEvent) => void,
         onError?: (event: ErrorEvent) => void,
-    ): KTX2Loader;
+    ): CompressedTexture;
```

Comment:
I think we can remove this method because it's inherited from `Loader`.

Reply:
Done